front/app/containers/Admin/dashboard/overview/index.tsx

```diff
@@ -164,84 +166,117 @@ class DashboardPageSummary extends PureComponent<PropsHithHoCs, State> {
-              infoMessage={infoMessage}
-              className="e2e-users-by-time-cumulative-chart"
-              {...this.state}
-            />
-            <LineBarChart
-              graphTitle={formatMessage(messages.inputs)}
-              graphUnit="ideas"
-              graphUnitMessageKey="ideas"
-              startAt={startAt}
-              endAt={endAt}
-              xlsxEndpoint={ideasByTimeCumulativeXlsxEndpoint}
-              className="e2e-ideas-chart"
-              lineStream={ideasByTimeCumulativeStream}
-              barStream={ideasByTimeStream}
-              {...this.state}
-            />
-            <LineBarChart
-              graphTitle={formatMessage(messages.commentsByTimeTitle)}
-              graphUnit="comments"
-              graphUnitMessageKey="comments"
-              startAt={startAt}
-              endAt={endAt}
-              xlsxEndpoint={commentsByTimeCumulativeXlsxEndpoint}
-              className="e2e-comments-chart"
-              lineStream={commentsByTimeCumulativeStream}
-              barStream={commentsByTimeStream}
-              {...this.state}
-            />
-            <Column>
-              <LineBarChartVotesByTime
-                className="fullWidth e2e-votes-chart"
+            <Box width="100%" display="flex">
+              <LineBarChart
+                graphUnit="users"
+                graphUnitMessageKey="users"
+                graphTitle={formatMessage(messages.usersByTimeTitle)}
                 startAt={startAt}
                 endAt={endAt}
+                xlsxEndpoint={usersByTimeXlsxEndpoint}
+                lineStream={usersByTimeCumulativeStream}
+                barStream={usersByTimeStream}
+                className="e2e-active-users-chart"
                 {...this.state}
               />
-              <SelectableResourceByProjectChart
-                className="dynamicHeight fullWidth e2e-resource-by-project-chart"
-                onResourceByProjectChange={this.onResourceByProjectChange}
-                resourceOptions={this.resourceOptions}
+              <BarChartActiveUsersByTime
+                graphUnit="users"
+                graphUnitMessageKey="activeUsers"
+                graphTitle={formatMessage(messages.activeUsersByTimeTitle)}
                 startAt={startAt}
                 endAt={endAt}
+                xlsxEndpoint={activeUsersByTimeXlsxEndpoint}
+                stream={activeUsersByTimeStream}
+                infoMessage={infoMessage}
+                className="e2e-users-by-time-cumulative-chart"
                 {...this.state}
               />
+            </Box>
+            <Title
+              ml="12px"
+              mt="40px"
+              width="100%"
+              variant="h2"
+              color="primary"
+              fontWeight="normal"
+            >
+              {formatMessage(overviewMessages.projectsAndParticipation)}
+            </Title>
+            <Column>
               <Outlet
```

Comment:
Not seeing any data here locally, is this intended?

![image](https://user-images.githubusercontent.com/13822005/202752485-cf717a20-a8d9-4aed-9b60-7dd03b63180d.png)